app/src/hooks/useCpk.tsx

```diff
@@ -1,5 +1,5 @@
 import CPK from 'contract-proxy-kit'
-import EthersAdapter from 'contract-proxy-kit/lib/cjs/eth-lib-adapters/EthersAdapter'
+import EthersAdapter from 'contract-proxy-kit/lib/esm/ethLibAdapters/EthersAdapter'
```

Comment:
Webpack actually uses the ESM versions when importing from 'contract-proxy-kit', so this change will make everything more consistent.